usr/src/uts/common/brand/lx/syscall/lx_fcntl.c

```diff
@@ -465,16 +475,32 @@ lx_fcntl_lock(int fd, int lx_cmd, void *arg)
 		if ((error = flock_check(vp, &bf, offset, maxoffset)) != 0)
 			break;
 
+		if (cmd == F_OFD_GETLK || cmd == F_OFD_SETLK ||
+		    cmd == F_OFD_SETLKW) {
```

Comment:
Need the comment as is shown in native fcntl.c support:
```
			/*
			 * TBD OFD-style locking is currently limited to
			 * covering the entire file.
			 */
```

Reply:
And Linux progs may not like that.  It'll be noteworthy, possibly even DTrace-probe-worthy.

Reply:
Inserted comments and probe into the appropriate spots.

Reply:
Now in #471